nanosv/1.2.4/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Base image
+FROM biocontainers/biocontainers:v1.2.0_cv1 
+
+################## METADATA ######################
+
+LABEL base_image="biocontainers:v1.2.0_cv1"
+LABEL version="1"
+LABEL software="nanosv"
+LABEL software.version="1.2.4"
+LABEL about.summary="NanoSV is a software package that can be used to identify structural genomic variations in long-read sequencing data"
+LABEL about.home="https://github.com/mroosmalen/nanosv"
+LABEL about.documentation="https://github.com/BioContainers/specs/wiki"
```

Comment:
This biocontainers doc page ;-) should siftware doc page, if any, else put homepage